package.json

```diff
@@ -91,6 +93,10 @@
     "webpack-dev-server": "^2.4.4"
   },
   "jest": {
+    "transform": {
+      "^.+\\.md?$": "markdown-loader-jest",
+      "^.+\\.jsx?$": "<rootDir>/node_modules/babel-jest"
```

Comment:
When `markdown-loader-jest` is added here, [the `babel-jest` transformer is no longer loaded automatically](https://facebook.github.io/jest/docs/en/configuration.html#transform-object-string-string), so we have to specify it manually.

![image](https://user-images.githubusercontent.com/8136030/36065188-39f6a73e-0e65-11e8-8bf3-5c2ef8239755.png)